src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationController.java

```diff
@@ -113,17 +128,15 @@ public String getConfirmation(@PathVariable String companyNumber,
 
     private String setUpPaymentDateDisplay(PayableLateFilingPenalty payableLateFilingPenalty) {
         if (payableLateFilingPenalty.getPayment() != null) {
-            return StringUtils.isEmpty(payableLateFilingPenalty.getPayment().getPaidAt()) ? "" :
-                    OffsetDateTime.parse(payableLateFilingPenalty.getPayment().getPaidAt(), DateTimeFormatter.ISO_OFFSET_DATE_TIME)
-                            .format(DateTimeFormatter.ofPattern("d MMMM uuuu", Locale.UK));
+            return LocalDate.now()
+                    .format(DateTimeFormatter.ofPattern("d MMMM uuuu", Locale.UK));
```

Comment:
@klai-ch -- is this correct? I know you added in the additional parsing for the date, and is the requirement just to pass back today's date (pretty sure it is).

Reply:
Yes, the requirement is passing back today's date. I use the payableLateFilingPenalty.payment.paidAt (which is also today's date) in order to make good use the module payableLateFilingPenalty.payment and I don't know if there is any chances that it is not pass today's date in the future, so I just think it is more accurate.